pkg/operator/options/options_validation.go

```diff
@@ -0,0 +1,67 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"knative.dev/pkg/apis"
+)
+
+func (o Options) Validate() (errs *apis.FieldError) {
```

Comment:
Minor thing, but we could probably replace the error type from `apis.FieldError` into just a standard error and merge the errors together with `mutlierr.Combine` and `multierr.Append`